### PR TITLE
Add volume and workload to French glossary

### PR DESCRIPTION
**Pull request by bcheronn**

### CODE REVIEW

**Comment by sftim** [comment]
@bcheronn please considering retitling
eg `/retitle add volume and workload to French glossary`

File: content/fr/docs/reference/glossary/volume.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- core-object
+- fundamental
+---
+ Un répertoire contenant des données, accessible aux {{< glossary_tooltip text="conteneurs" term_id="container" >}} d'un {{< glossary_tooltip term_id="pod" >}}.
+
+<!--more--> 
+
+Un volume Kubernetes vit aussi longtemps que le pod qui le contient. Par conséquent, un volume survit à tous les conteneurs qui s'exécutent dans le pod, et les données contenues dans le volume sont préservées lors des redémarrages du conteneur.
+
+Voir [storage](https://kubernetes.io/docs/concepts/storage/) pour plus d'informations.
```
Comment (sftim) [inline content/fr/docs/reference/glossary/volume.md:20]:
```suggestion
Voir [stockage](https://kubernetes.io/docs/concepts/storage/) pour plus d'informations.
```

File: content/fr/docs/reference/glossary/workload.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Workload
+id: workloads
+date: 2019-02-13
+full_link: /docs/concepts/workloads/
```
Comment (sftim) [inline content/fr/docs/reference/glossary/workload.md:5]:
```suggestion
full_link: /fr/docs/concepts/workloads/
```

File: content/fr/docs/reference/glossary/volume.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: Volume
+id: volume
+date: 2018-04-12
+full_link: /docs/concepts/storage/volumes/
```
Comment (sftim) [inline content/fr/docs/reference/glossary/volume.md:5]:
```suggestion
full_link: /fr/docs/concepts/storage/volumes/
```

**Comment by bcheronn** [comment]
Suggestions applied

**Comment by bcheronn** [comment]
/retitle Add volume and workload to French glossary

**Comment by sftim** [comment]
It's all good to me. I'll leave approval to the French localization team.

**Pull request merged**